src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -210,6 +229,10 @@ export class DotEditContentComponent implements OnInit, OnDestroy {
             });
     }
 
+    private pageServiceSave(): Observable<string> {
+        return this.dotEditPageService.save(this.pageState.page.identifier, this.dotEditContentHtmlService.getContentModel());
```

Comment:
Please double check the response of the endpoint because I think the response is not an ```Observable<string>``` anymore.